app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -10,42 +10,26 @@ import android.view.Menu
 import android.view.MenuInflater
 import android.view.MenuItem
 import android.view.View
-import android.widget.EditText
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.Toolbar
-import androidx.core.widget.doOnTextChanged
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.lifecycleScope
 import androidx.navigation.Navigation
-import kotlinx.android.synthetic.main.fragment_edit_bookmark.bookmarkNameEdit
-import kotlinx.android.synthetic.main.fragment_edit_bookmark.bookmarkParentFolderSelector
-import kotlinx.android.synthetic.main.fragment_edit_bookmark.bookmarkUrlEdit
-import kotlinx.android.synthetic.main.fragment_edit_bookmark.bookmarkUrlLabel
+import androidx.navigation.fragment.findNavController
+import kotlinx.android.synthetic.main.fragment_edit_bookmark.*
+import kotlinx.coroutines.*
```

Comment:
nit - we don't use star imports besides for kotlin synthetics.
Your autoformatting may need to be tweaked. In Android Studio -> Preferences -> Editor -> Code Style -> Kotlin , make sure `use single name import` is checked

Reply:
I made these changes in my editor options and re-imported this, but still got the star import.